utils/flink-components-testkit/src/main/scala/pl/touk/nussknacker/engine/flink/util/test/FlinkTestScenarioRunner.scala

```diff
@@ -87,11 +87,14 @@ class FlinkTestScenarioRunner(
     WaitForJobStatusPatience.interval * 2
   )
 
-  override def runWithData[I: ClassTag, R](scenario: CanonicalProcess, data: List[I]): RunnerListResult[R] = {
+  override def runWithData[I: ClassTag, R](
+      scenario: CanonicalProcess,
+      data: List[I]
+  ): RunnerListResult[R] = {
     runWithTestSourceComponent(
       scenario,
       NodesDeploymentData.empty,
-      ProcessVersion.empty.copy(processName = scenario.metaData.name),
+      labels = List.empty,
```

Comment:
I replaced `ProcessVersion` with labels list, because `ProcessVersion` is part of API that I want to untangle someday. It contains a lot of strange things, not related with each others. I constantly reduce number of occurrences of this class. It is not related with this change.